### PR TITLE
Serial Correlation

### PR DESCRIPTION
**Pull request by tjschechter**
This is a contribution for serial/autocorrelation in the LOST repo.

### CODE REVIEW

**Comment by NickCH-K** [comment]
This looks good, nice and simple. Will probably take me a while to get to it - a bunch of PRs just came in all at once, but it will be up before too long.

**Comment by grantmcdermott** [comment]
@tjschechter I see that you've changed the actual `NewPageTemplate.md` file. 

This template needs to be _copied_ and renamed (e.g. `Serial_Correlation.md`), rather than rewritten. You can then save this file under the appropriate directory, i.e. [Time_Series](https://github.com/LOST-STATS/lost-stats.github.io/tree/source/Time_Series).

@NickCH-K might have a different opinion. But I think the easiest way for you to do this --- rather than rewinding your git history --- is going to be the following: Close this PR and then create a new PR with the correct `Time_Series/Serial_Correlation.md` file plus any changes that you want to make. (TL;DR please make sure that you don't commit any changes to the NewPageTemplate.md file.)

**Comment by NickCH-K** [comment]
Oh good point. Agreed, that's the best way to fix it.

**Comment by khwilson** [comment]
Fixed it directly :-)

**Comment by khwilson** [comment]
For reference, the series of commands was:

```
git checkout source
git pull --rebase origin source
gh pr checkout 143
git mv NewPageTemplate.md Time_Series/Serial_Correlation.md
git checkout source NewPageTemplate.md
git push https://github.com/tjschechter/lost-stats.github.io.git HEAD:source
```

This commit brought to you by the alt text of [this XKCD](https://xkcd.com/1597/) comic.

**Comment by NickCH-K** [comment]
Great! Made changes (for next time please be sure your code example runs before submitting). Added Stata and Python too.
